mercury/models.py

```diff
@@ -1,38 +1,62 @@
 from django.db import models
 
 
-class SimulatedData(models.Model):
-    """This model represents the sensors that we expect to
+class TemperatureSensor(models.Model):
+    """This model represents the Tempeature sensor that we expect to
     potentially available in the future in the NYU Motorsports
```

Comment:
to _be_ potentially available (and please update the same as other docstrings in the other classes).